test/TestCase.h

```diff
@@ -31,6 +31,43 @@
 namespace solidity::frontend::test
 {
 
+/**
+ * A reader for test case data file, which parses source, settings and (optionally) simple expectations.
+ */
+class TestCaseReader
+{
+public:
+	TestCaseReader() = default;
+	TestCaseReader(std::string const& _filename);
+
+	std::map<std::string, std::string> sources() { return m_sources; }
+	std::string source();
+	std::size_t lineNumbers() { return m_lineNumbers; }
```

Comment:
Why plural if it only returns a single value?

Reply:
Purely for a historical reason. Changed, 6c9736b.